### ISSUE TITLE
Wildcard (*) character not working

### ISSUE THREAD

**Issue by fancymans** [rodi01/RenameIt#61]
Seems like I can no longer rename layers with the wildcard character as "match all after this point."

**For example, I have the following layers:**
Rect A
Rect B
Rect C

**Try to rename via Find & Replace**
Find: Rect*
Replace With: Circle %A

**Expected Results:**
Circle A
Circle B
Circle C

**Actual Results:**
Rect A
Rect B
Rect C

Nothing happens. I remember this working in the past, but it doesn't work anymore.

**Comment by rodi01** [comment]
Hi @fancymans,

The keywords were never available in the find and replace view. They are only available on the rename view. It's something that I'm considering adding in later releases. 

One change that I've made was that the * wildcard now needs to be precessed by %. So %* should get the current layer's name in the rename view.

**Comment by fancymans** [comment]
I see... maybe I have faulty memory. 🤔

Anyways thanks for the explanation. I suppose it’d be a feature request or enhancement to get the functionality I was looking for. 

Cheers!

**Comment by fancymans** [comment]
Just wanted to follow up on this issue as I noticed in the .gif you posted that the functionality I described in the original post seems to be shown. 

https://github.com/rodi01/RenameIt/blob/master/docs/static/img/gifs/artboard_rename.gif

In this gif, you show using * as a copy of the original layer name. Now I'm still not sure if it's just a copy of the original layer/artboard name or if it has wildcard properties, but does that mean this is a WIP feature for a future release?
